ddtrace/contrib/django/patch.py

```diff
@@ -663,6 +663,46 @@ def django_asgi_modifier(span, scope):
 
 
 class _DjangoUserInfoRetriever(_UserInfoRetriever):
+    def __init__(self, user, credentials=None):
+        self.credentials = credentials if credentials else {}
+        super(_DjangoUserInfoRetriever, self).__init__(user)
+
+    def user_exists(self):
+        if self.user:
+            return True
+
+        if not self.credentials:
+            return False
```

Comment:
It's not the same if we don't have the credentials than the user does not exist. Maybe this function could return `None` instead of `False` if we don't know? AFAIK it's very important that we are sure while sending these events.

Reply:
Done